### ISSUE TITLE
Blank files after updating to 1.6.1

### ISSUE THREAD

**Issue by nicholas-c** [atom/atom#11327]
### Prerequisites
* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Key Info
**OS:** Windows 10
**Pre-update version:** Atom 1.6.0
**Post-update version:** Atom 1.6.1
**Update method:** Auto-update

Spoken to @thomasjo in slack channel (Recommended posting here), opening files on my PC are fine, network drive issues, SAMBA mounted network drive in windows, no issues before this.

Coming back to work on Tuesday atom had auto-updated, I believe to 1.6.1. This caused issues where the application would hang for 30-40 seconds upon basically any action, maybe get as far as clicking a button for it to hang again.

As you can imagine, hard to develop anything! So I completely uninstalled everything, removed all traces from %appdata% and my user folder (.atom - This is backed up) - I reinstalled a fresh copy of 1.6.1 from the site, everything seemed to be working, until I added a project folder and opened a file... Blank. Nothing, Nada. Even stranger, if I then close atom, relaunch, close the file and undo the close tab, it has it's contents... But no other file does.

Again, after this I removed all traces again and installed 1.6.0 with no luck resolving the issue, followed by trying out the 1.7.0 beta 3.

**Errors in developer console:**
```Error refreshing repository status:(anonymous function) @ git-repository-async.js:971
git-repository-async.js:972 Error: Failed to resolve path '\\fs1\Propeller Files\Clients\rockliffe-drupal.com\versions\nick\httpdocs\sites\all\modules\examples': The system cannot find the file specified.
(…)(anonymous function) @ git-repository-async.js:972
git-repository-async.js:971 Error refreshing repository status:(anonymous function) @ git-repository-async.js:971
git-repository-async.js:972 Error: Failed to resolve path '\\fs1\Propeller Files\Clients\rockliffe-drupal.com\versions\nick\httpdocs\sites\all\modules\gauth': The system cannot find the file specified.
(…)(anonymous function) @ git-repository-async.js:972
git-repository-async.js:971 Error refreshing repository status:(anonymous function) @ git-repository-async.js:971
git-repository-async.js:972 Error: Failed to resolve path '\\fs1\Propeller Files\Clients\rockliffe-drupal.com\versions\nick\httpdocs\sites\all\modules\shib_auth': The system cannot find the file specified.
(…)```

**Comment by thomasjo** [comment]
I haven't attempted to reproduce this on a Windows VM yet, but I suspect this is either caused by the space in the path, or related to the fact the file is stored on a network drive.

My assumption here might be faulty, but I'm pretty sure this is caused by either #11277 or #11295 (or a combination of the two). That's essentially all that was [changed](https://github.com/atom/atom/compare/v1.6.0...v1.6.1) from v1.6.0 to v1.6.1.

/cc @joshaber

**Comment by joshaber** [comment]
Entirely possible. Those console errors don't jump out at me as something that'd prevent Atom from displaying anything though.

**Comment by jamiewade** [comment]
I was also having the exact same problem, double clicking on files in the tree opened them in a new tab but the files were completely empty and Atom would hang. I am also running Atom 1.6.1 on Windows 10.

> ... I suspect this is either caused by the space in the path, or related to the fact the file is stored on a network drive.

For each of my development projects (in the Project Manager package) I have two folders; my local working copy on my `C:` drive and a shared development site copy on a network drive.

The minute I removed the shared network drive folder from my project, Atom started working perfectly again. Hope that helps somehow :smile:

**Comment by Rondo5000** [comment]
I am experiencing this issue on Ubuntu 14.04 with an SSH mounted drive, and I could guess that it is related to changes brought about to address #11214, which had fixed issue of the GUI freezing for me, but not the files themselves actually loading when chosen, leaving it unfortunately still unusable.  And my apologies if posting the details in there (it being closed) broke etiquette, btw.

**Comment by dkt1412** [comment]
I'm also experiencing this issue, BUT I'm having this issue on a **mac (os x 10.11.4)**.  I've only noticed this issue with files on mounted drive (via SSHFS).

I'm able to consistently reproduce this issue with Atom with the following steps:
1. Restart macbook.
2. Mount Ubuntu 14.04 drive via SSHFS
3. `atom --safe <mount drive>`
4. Open some files
5. Sleep laptop such that mount **disconnects**
6. Wake laptop and reconnect mount
7. Reopen/reload atom editor.
8. Open files -> Files will be empty/blank.

I am running **Atom (v 1.6.2)**, and running step 7 and 8 in Safe Mode also produces the same issues.  I've also tried removing all of Atom's caches (storage, compile-cache, Library/<atom related files>) and tried reinstalling Atom as well.  None of these methods were able to get me out of this state.  So far, only a restart seems to have worked.

Based on this issue, I've tried back-tracking on all the releases and seeing which ones would finally work for me.  It seems the latest Atom release that would let me successfully open a file on the mounted drive is 1.5.4, which probably makes sense given the fixes that went into the 1.6* releases.  

My experience on the following version:
1. 1.6.1-2: This issue
2. 1.6.0: UI freezes (gives me "Keep Waiting/Close" prompt) when I open a mounted file
3. 1.5.4: UI doesn't freeze and file content shows up when it's opened!

Hope this helps/please let me know if I can provide any more details!

**Comment by matteocng** [comment]
I'm experiencing the same issue on OS X 10.11.3, the editor is unusable with SSHFS mounted filesystems.

I can confirm @dkt1412 testing results:
> My experience on the following version:
1. 1.6.1-2: This issue
2. 1.6.0: UI freezes (gives me "Keep Waiting/Close" prompt) when I open a mounted file
3. 1.5.4: UI doesn't freeze and file content shows up when it's opened!

**Comment by tshazar** [comment]
@dkt1412 great description of the issue. I see the same (on OS X 10.11.4) with smb/cifs mounted drives.

My experience on the following version:
1. 1.6.1-2: This issue
2. 1.6.0: UI freezes (gives me "Keep Waiting/Close" prompt) when I open a mounted file
3. 1.5.4: UI doesn't freeze and file content shows up when it's opened!

**Comment by birkoss** [comment]
Same thing happen here with sshfs, mostly on big folders using GIT, I can fix this by renaming my .git subfolder to something else (like test.git/) and every files open normally.

**Comment by abudayah** [comment]
I can't work any more on atom on windows because of this issue :disappointed:

**Comment by Rondo5000** [comment]
Yes @abudayah, this branch of issues (which includes ticket 11214 as I stated above) are unfortunately showstoppers.  Keep in mind that you can always downgrade to version 1.5.4 in the meantime which does not suffer from these regressions:

https://github.com/atom/atom/releases/tag/v1.5.4

**Comment by abudayah** [comment]
@Rondo5000 Thanks, and btw when I disable git folder back to work normal in v1.6.2

**Comment by nicholas-c** [comment]
@abudayah Going back to 1.5.4 gives you a working editor, and you get to keep the git stuff

**Comment by dymoksc** [comment]
I have the same issue over sshfs with GIT on ubuntu, renaming .git won't help

**Comment by tshazar** [comment]
Looks like we have all supported OSes represented on this thread can we have the owner of this ticket update to reproduced on all OSes. Then we may get some traction... @nicholas-c

**Comment by nicholas-c** [comment]
@tshazar - I can't change the labels sadly, not for me to label as I've only posted an issue and it's down to the repo contributors to label and task the issue.

@thomasjo may be able to reassign the labels but "needs-reproduction" with be reproduction by the development team.

**Comment by Zireael07** [comment]
@nicholas-c: But you can change the issue title!

**Comment by thomasjo** [comment]
> may be able to reassign the labels but "needs-reproduction" with be reproduction by the development team.

Correct! The `needs-reproduction` label means that one of the maintainers need to reliably reproduce the issue.

**Comment by tshazar** [comment]
FYI ... still reproducible on OSX 10.11.4 with atom v1.7.0 (2016-04-13)

Looks like it is back to v1.5.4 :(

**Comment by jwahl** [comment]
Having same problem - going back to 1.5.4 for me too.

**Comment by rushl** [comment]
Issue occurs for me too. I've updated up to 1.7.0, still no ability to access files on a network drive.

**Comment by Rondo5000** [comment]
The blank files issue (as far as I can tell) appears to be resolved in 1.8.0-beta0, and likely due to #11315.  I'm still experiencing a few issues with git project folders taking a while to 'index' the git information correctly compared to 1.5.4 -- i.e. 1.8.0-beta0 may take a couple of minutes with a larger remote (but still LAN, I'll note) project folder before a community package such as "line-diff-details" has the info it needs to begin highlighting project changes -- but as far as blank files and freezes and whatnot it seems to be very stable.

My huge thanks go toward @joshaber in particular for working on this branch of issues.

**Comment by weboide** [comment]
Same here, I had to upgrade to 1.8.0-beta1 to finally be able to open files over SSHFS. Otherwise I would have gotten blank files.

**Comment by dymoksc** [comment]
Confirming - in 1.8.0-beta2 it works again :)

**Comment by 50Wliu** [comment]
Closing as this issue appears to be resolved in Atom 1.8.0.

**Comment by Queatz** [comment]
I'm still getting this on 1.8.0.  Opening files from sshfs directory shows empty files.
